prow/getting_started_deploy.md

```diff
@@ -312,21 +312,30 @@ gcloud iam service-accounts keys create --iam-account "${identifier}" service-ac
 kubectl -n test-pods create secret generic gcs-credentials --from-file=service-account.json # step 6
 ```
 
-Before we can update plank's `default_decoration_config` we'll need to know the version we're using
+### Configure the version of plank's utility images
+
+Before we can update plank's `default_decoration_config` we'll need to retrieve the version of plank using the following:
+
 ```sh
 $ kubectl get pod -lapp=plank -o jsonpath='{.items[0].spec.containers[0].image}' | cut -d: -f2
-v20190619-25afbb545
+v20191108-08fbf64ac
 ```
+Then, we can use that tag to retrieve the corresponding utility images in `default_decoration_config` in `config.yaml`:
+
+By doing this, your are pinning yourself to specific, known good-versions of these images.
```

Comment:
nit: These tags aren't really known to be "good-versions". What we are ensuring is that `plank` and the utility images are using the same version.

Reply:
@cjwagner Thank you for this clarification, I'll amend accordingly.